src/keyboard/gutter_handler.js

```diff
@@ -205,6 +206,7 @@ class GutterKeyboardHandler {
             switch (this.activeLane){
                 case "fold":
                     this.$blurFoldWidget(this.activeRowIndex);
```

Comment:
Does `$blurFoldWidget` here expect there always to be a foldwidget at the provided row or will it handle the possibility of no fold widget being there okay?